improver_tests/acceptance/test_spot_extract.py

```diff
@@ -75,6 +86,8 @@ def test_lapse_rate_adjusted_uk(tmp_path):
         "--output",
         output_path,
         "--apply-lapse-rate-correction",
+        "--new-title",
+        UK_SPOT_TITLE,
```

Comment:
Yes, that's what's happening.

For what it's worth, I don't think handling spot titles via command line is the right way forward, as it can lead to inconsistencies such as these (which on the face of it don't appear to make much sense).  But perhaps that can be revisited some time in the future.